modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TreeTableRowSkinTest.java

```diff
@@ -208,14 +209,15 @@ public void treeTableRowWithCellSizeShouldHonorPadding() {
 
     @Test
     public void treeTableRowWithFixedCellSizeShouldIgnoreVerticalPadding() {
-        int top = 10;
-        int right = 20;
-        int bottom = 30;
-        int left = 40;
+        int top = 11;
+        int right = 23;
+        int bottom = 37;
+        int left = 49;
```

Comment:
After the fix and test change, this test now fails for me.

Reply:
I'm not quite seeing why these changes are needed.  How are they related to the memory leak?

Reply:
They shouldn't be needed. This points to a problem either with the fix, or a preexisting problem with this test.